### PR TITLE
elif correction

### PR DESCRIPTION
**Pull request by coffeepac**

### CODE REVIEW

File: max_k8s_version.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ OUTFILE=$1
 if [ -z ${OUTFILE} ]; then 
   echo "must specify a file to save the max version too"
```
Comment (DStorck) [inline max_k8s_version.sh]: too?

**Comment by davidewatson** [comment]
CI failed because the etcd cluster failed to form:

> [aws] FAILED - RETRYING: Wait for etcd cluster to form (1 retries left).
[aws] fatal: [localhost]: FAILED! => {"attempts": 30, "changed": true, "cmd": ["ssh", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null", "-F", "/home/jenkins/workspace/K2_PR-694-BFT7YJ63M6ACLYQ4C2KP4JAWU44Y53AFYFS2M4SRS2MIVPCWBJOA/cluster/aws//pr-694-1/ssh_config", "etcdEvents-1", "etcdctl", "--ca-file=/etc/etcd/ssl/client-ca.pem", "--cert-file=/etc/etcd/ssl/client.pem", "--key-file=/etc/etcd/ssl/client-key.pem", "--endpoints=https://127.0.0.1:2381", "cluster-health"], "delta": "0:00:01.483752", "end": "2017-08-01 06:05:13.323456", "failed": true, "rc": 4, "start": "2017-08-01 06:05:11.839704", "stderr": "Warning: Permanently added 'ec2-18-220-66-73.us-east-2.compute.amazonaws.com,18.220.66.73' (ECDSA) to the list of known hosts.\r\nError:  client: etcd cluster is unavailable or misconfigured\nerror #0: client: endpoint https://127.0.0.1:2381 exceeded header timeout", "stderr_lines": ["Warning: Permanently added 'ec2-18-220-66-73.us-east-2.compute.amazonaws.com,18.220.66.73' (ECDSA) to the list of known hosts.", "Error:  client: etcd cluster is unavailable or misconfigured", "error #0: client: endpoint https://127.0.0.1:2381 exceeded header timeout"], "stdout": "cluster may be unhealthy: failed to list members", "stdout_lines": ["cluster may be unhealthy: failed to list members"]}

This is the same problem I've seen on a few other etcd PRs I have in flight.

File: max_k8s_version.sh [hunk 2]
```diff
@@ -12,8 +12,9 @@ my_dir=$(dirname "${BASH_SOURCE}")
 
 OUTFILE=$1
 if [ -z ${OUTFILE} ]; then 
-  echo "must specify a file to save the max version too"
+  echo "must specify a file too save the max version too"
```
Comment (davidewatson) [inline max_k8s_version.sh]: I think @DStorck meant for this to read `echo "must specify a file to save the max version to"`, or similar.

Comment (DStorck) [inline max_k8s_version.sh]: Yes, I thought it should have read "must specify a file to save to save the max version to"

**Pull request merged**